app/lib/screens/HomeScreens/ChatScreen.dart

```diff
@@ -295,122 +295,79 @@ class _ChatScreenState extends State<ChatScreen> {
-                      '${snapshot.requireData.length.toString()} Members',
-                      style: GoogleFonts.montserrat(
-                        color: AppColors.primaryColor,
-                        fontSize: 12,
-                        fontWeight: FontWeight.w300,
-                      ),
-                    );
-                  } else {
-                    return Container(
-                      height: 15,
-                      width: 15,
-                      child: CircularProgressIndicator(
-                        color: AppColors.primaryColor,
-                      ),
-                    );
-                  }
-                },
-              ),
-            ],
-          ),
+        leading: Row(
+          children: <Widget>[
+            IconButton(
+              onPressed: () {
+                Navigator.pop(context);
+              },
+              icon: SvgPicture.asset('assets/images/back_button.svg'),
+            ),
+          ],
         ),
-        actions: [
-          // SizedBox(
-          //   width: 70,
-          // ),
-          Padding(
-            padding: const EdgeInsets.only(right: 10),
-            child: FutureBuilder<Uint8List>(
-              future: widget.room.avatar().then((fb) => fb.asTypedList()),
-              // a previously-obtained Future<String> or null
-              builder:
-                  (BuildContext context, AsyncSnapshot<List<int>> snapshot) {
-                if (snapshot.connectionState == ConnectionState.waiting) {
-                  return SizedBox(
-                    height: 20,
-                    width: 20,
-                    child: CircularProgressIndicator(
+        title: Column(
+          mainAxisSize: MainAxisSize.max,
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            FutureBuilder<String>(
+              future: widget.room.displayName(),
+              builder: (BuildContext context, AsyncSnapshot<String> snapshot) {
+                if (snapshot.hasData) {
+                  return Text(
+                    snapshot.requireData,
+                    overflow: TextOverflow.clip,
+                    style: TextStyle(
+                      fontSize: 14,
+                      fontWeight: FontWeight.w600,
+                      color: Colors.white,
+                    ),
+                  );
+                } else {
+                  return Text('Loading Name');
+                }
+              },
+            ),
+            SizedBox(height: 5),
+            FutureBuilder<FfiListMember>(
+              future: widget.room.activeMembers(),
+              builder: (
+                BuildContext context,
+                AsyncSnapshot<FfiListMember> snapshot,
+              ) {
+                if (snapshot.hasData) {
+                  return Text(
+                    '${snapshot.requireData.length.toString()} Members',
+                    style: GoogleFonts.montserrat(
```

Comment:
I think we moved away from `montserrat` with the google licence PR...

Reply:
I was wondering why google fonts giving me errors. I thought its updated in merged PR